dashboard/test/ui/features/learning_platform/projects/starwars_project.feature

```diff
@@ -15,7 +14,10 @@ Scenario: Starwars Flow
   And I click selector ".project_save"
   And I wait until element ".project_edit" is visible
   Then I should see title "Code Ninja III: Revenge of the Semicolon - Play Lab"
-  And I press "runButton"
+  # There used to be a 'And I press "runButton"' step here but it causes the test
```

Comment:
Is it necessary to leave this comment?

Reply:
i second that question. not sure this comment is useful going forward.

Reply:
Ok, I'll just remove it, thanks!